packages/expo-router/src/modal/Modal.tsx

```diff
@@ -30,6 +31,13 @@ export interface ModalProps extends ViewProps {
    * Callback that is called after modal is shown.
    */
   onShow?: () => void;
+  /**
+   * Callback that is called after detent change.
+   * Works only when `presentation` is set to `formSheet`.
+   *
+   * Does not work on web.
```

Comment:
Could work on web though, Vaul has APIs for it that we could use.
Platform comments please with

`@platform ios`
`@platform android`

One per line

Reply:
Yeah I tried looking for it in Vaul docs but couldn't find direct support. Unless I'm missing something 🤔